sotodlib/coords/helpers.py

```diff
@@ -433,3 +433,67 @@ def _apply_inverse_weights_map(inverse_weights, target):
         target.shape[1], target.shape[2], target.shape[0], 1)
     m1 = np.matmul(iw, m)
     return m1.transpose(2,3,0,1).reshape(target.shape)
+
+
+class ToastishQuat(np.ndarray):
```

Comment:
Is there a reason this needs to be a class?  It seems like it would be simpler to just have two small functions to convert back and forth between the formats- especially since it is only being used in one place, to convert back and forth.  If you still want to have a class that inherits from ndarray, can we at least change name?  We are actively working to support both Boost and SciPy format quaternions in toast, so in the very near future this setting will be toggled in `sotodlib.toast.__init__.py` once for the whole package.

For functions maybe `quat_boost_to_scipy()` / `quat_scipy_to_boost()` or `quat_1ijk_to_ijk1()` / `quat_ijk1_to_1ijk()`. Or if we must have a class, can we change it to `ToastQuat` or maybe `ScipyQuat`, `ScalarLastQuat`, etc.